docs/product/base/colors.html

```diff
@@ -599,7 +599,7 @@
 <p>…</p>
 <p class="fc-dark">…</p>
 {% endhighlight %}
-        <div class="stacks-preview--example d-flex fd-column gs16 gsy my0 fs-body2">
+        <div class="stacks-preview--example d-flex fd-column gx16 my0 fs-body2">
```

Comment:
## No action necessary

This renders slightly differently than what's in `develop`.

<details>
<summary>develop</summary>

![image](https://user-images.githubusercontent.com/647177/197639453-4dd1dfde-37dd-4d2e-8ac1-e8b852e867be.png)

</details>

<details>
<summary>This PR</summary>

![image](https://user-images.githubusercontent.com/647177/197639496-a9e032fa-738a-4f3d-b64a-4c0a60c0e463.png)

</details>

This is a result of the newly applied `.gx16` class no longer overriding margin on the child `<p>` elements. Previously, the margin hack from `.gs16.gsy` replaced any default margins set on `<p>`. IMO this is fine in the context of docs, but we'll want to be conscious of how this might affect similar code in production.

Reply:
Good catch! I am not entirely sure why we use Flex for these paragraphs to be honest. I just left the paragraphs and removed flex all together in my local machine and it looks pretty much the same to the original version. I will update the PR.